USAGE.md

```diff
@@ -1,370 +1,707 @@
+We also recommend reading the [ATT&CK Design and Philosophy Paper](https://attack.mitre.org/docs/ATTACK_Design_and_Philosophy_March_2020.pdf), which describes high-level overall approach, intention, and usage of ATT&CK.
 
-The above STIX types are found as literal strings assigned to the `type` property of the STIX JSON object. As shown in the table, in STIX 2.0, there are objects called "Course(s) of Action" used to describe mitigations to ATT&CK techniques. Similarly, the STIX 2.0 object called "Attack Pattern" describes techniques, etc. It should also be noted that Tactics are not an explicit object type in STIX 2.0, and they are referenced implicitly as kill chain phases within the other object types, as described in the tables below.
+# The ATT&CK Data Model
 
-## Properties
-The following is a table mapping of ATT&CK properties, the old ATT&CK MediaWiki names, and the new STIX properties. Some of these properties are standard STIX properties, while others were custom-created for compatibility with ATT&CK. These properties are accessed from STIX objects as JSON properties.
+The data in this repository is STIX2.0 and divided into three folders, one for each domain of ATT&CK. These domains generally follow the same format with a few departures. Domain differences will be noted in the relevant sections of this document. 
 
-### Migrating from MediaWiki
+STIX uses a different terminology than ATT&CK. The following table is a mapping of ATT&CK concepts to STIX 2.0 objects:
 
-### Common properties (on all objects)
-ATT&CK Property | ATT&CK MediaWiki | STIX Properties
---------------- | ---------------- | ---------------
-**Entry ID**    | `Has ID` | `external_references[i].external_id` where `external_references[i].source_name == "mitre-attack"`
-**Entry URL**   | `URL` | `external_references[i].url` where `external_references[i].source_name == "mitre-attack"`
-**Entry Title** | `Has display name` | `name`
-**Entry Text**  | `Has description` | `description`
-**Citation**    | `Citation reference` | `external_references`
-**Deprecated**  | `Deprecated` | `x_mitre_deprecated`
-**Revoked**     | `Not available via MediaWiki API` | `revoked`
-**Old ATT&CK ID** | `Not available via MediaWiki API` | `x_mitre_old_attack_id`
+| ATT&CK concept | STIX object type |
+|:------------|:----------|
+| [Matrix](#matrices)              | `x-mitre-matrix` |
+| [Tactic](#tactics)               | `x-mitre-tactic` |
+| [Technique](#techniques)         | `attack-pattern` |
+| [Sub-technique](#sub-techniques) | `attack-pattern` where `x_mitre_is_subtechnique = true` |
+| [Procedure](#procedures)         | `relationship` where `relationship_type = "uses"` and `target_ref` is an `attack-pattern` |
+| [Mitigation](#mitigations)       | `course-of-action` |
+| [Group](#groups)                 | `intrusion-set`  |
+| [Software](#software)            | `tool` or `malware` |
 
+Two additional object types are found in the ATT&CK catalog:
+ 
+| STIX object type | About |
```

Comment:
Is this saying these are predefined STIX object types (like the right side of the table above) that are included in ATT&CK and stays the same, or is it saying these are ATT&CK concepts (left side of table above) that don't map to STIX 2.0 (confused by the `STIX object type` column header).

Reply:
I think this makes more sense with Jack's new explanation sentence:

> ATT&CK uses a mix of predefined and custom STIX objects to implement ATT&CK concepts. 

I think this helps make it more obvious that the lefthand side is the concept and the righthand side is how it's implemented.